website/server/models/user/schema.js

```diff
@@ -24,7 +24,21 @@ let schema = new Schema({
     local: {
       email: {
         type: String,
-        validate: [validator.isEmail, shared.i18n.t('invalidEmail')],
+        validate: [{
+          validator: validator.isEmail,
+          msg: shared.i18n.t('invalidEmail'),
+        }, {
+          validator: function notEmailDomains (email) {
+            let domains = ['habitica\\.com', 'habitrpg\\.com'];
```

Comment:
Good point, I had intended to make it:

```js
return INVALID_DOMAINS.every(domain => email.indexOf(`@${domain}`) === -1);
```

And just forgot. That should take care of that edge case.